### PR TITLE
Update publish popup title when project is live - Issue 1918 

### LINKED ISSUE #1918: Change Publish popup title when project is Live
Currently, the popup title always says "Publish your Project", even if it is already published...

image

Let's change it to "Your Project is Online" when the project is published.

### PR DESCRIPTION
**Pull request by marcobeltempo**
Fixes #1918 
The original issue was that the publish popup title would always display **"Publish your Project"** regards if it was live or not.

@flukeout @humphd My solution was to reuse the existing `h1`  in "_views/editor/publish.html_" to assign the value depending on the current state of the project performed on the server side. 
Unpublished title = **"Publish your Project"**  
Published title = **" Your Project is Online"**   

1. created var `TEXT_PUBLISHED_HEADER = strings.get("publishedHeader");` holds the string value from 
_"locales/en-US/server.properties"_

2. added publishHeader to the Publisher() object in "\public\editor\scripts\ui\publisher.js". This gets the `<h1>` id in _"views/editor/publish.html"_
 
3.  `Publisher.prototype.updateDialog` located in  _"views/editor/publish.html"_ already checks the handler of the project status.

4. If the project is published,  the header gets assigned `TEXT_PUBLISHED_HEADER` from step 1

5. If the project is unpublished the header is set to default

thimble_issue1918_fix_capture

### CODE REVIEW

File: locales/en-US/server.properties [hunk 1]
```diff
@@ -213,6 +213,7 @@ snippetJSChangeStyleComment=Select the element with id='alert'
 
 # Publishing
 publishHeader=Publish your Project
+publisedhHeader=Your Project is Online
```
Comment (gideonthomas) [inline locales/en-US/server.properties]: since you're using this in the client files you should add this string to the client.properties file instead as per https://github.com/mozilla/thimble.mozilla.org/wiki/Localization#client-side-localization.
Also rename this to `publishHeaderOnline`

**Comment by humphd** [comment]
@marcobeltempo before we go to land this, I need to show you how to rebase your work, since it looks like you've got a bunch of commits in here that aren't part of your patch (did you merge with master, maybe?).  It's not a problem, we can fix it, but FYI, you don't need to do that while you're developing.

**Comment by marcobeltempo** [comment]
@humphd that would be great! I made a mess somewhere down the line and a bigger one trying to fix it.

**Comment by humphd** [comment]
Yeah, it's easily fixed, don't worry.  For now, just keep getting your PR fixed.  Then I'll give you a hand.  Remind me please, I'll forget, but I need to cover this with all of you.

File: locales/en-US/client.properties [hunk 2]
```diff
@@ -27,6 +27,7 @@ renameProjectSavingIndicator=Saving…
 publishPublishingIndicator=Publishing…
 publishUnpublishingIndicator=Unpublishing…
 publishDeleteWarning=Are you sure? The current link will stop working and if you publish again, the link will be different.
+publishHeaderOnline=Your Project is Online
```
Comment (gideonthomas) [inline locales/en-US/client.properties]: also move the `publishHeader` string from `server.properties` to `server-client-shared.properties` since we are going to use that string in both the server and the client.

File: public/editor/scripts/ui/publisher.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ var TEXT_UNPUBLISH = strings.get("publishDeleteBtn");
 var TEXT_UNPUBLISHING = strings.get("publishUnpublishingIndicator");
 var TEXT_UPDATE_PUBLISH = strings.get("publishChangesBtn");
 var TEXT_UNPUBLISH_WARNING = strings.get("publishDeleteWarning");
+var TEXT_PUBLISHED_HEADER = strings.get("publishHeaderOnline");
```
Comment (gideonthomas) [inline public/editor/scripts/ui/publisher.js]: let's call the `TEXT_PUBLISH_HEADER_ONLINE`
and add another variable here called `TEXT_PUBLISH_HEADER` that stores the `publishHeader` string

File: public/editor/scripts/ui/publisher.js [hunk 4]
```diff
@@ -319,6 +321,7 @@ Publisher.prototype.updateDialog = function(publishUrl, allowUnpublish) {
   // "publish"/"cancel" buttons
   if (allowUnpublish) {
     this.dialog.buttons.parent.addClass("hide");
+    this.dialog.publishHeader.text(TEXT_PUBLISHED_HEADER);
```
Comment (gideonthomas) [inline public/editor/scripts/ui/publisher.js]: you also need to set the title in the else condition back to `TEXT_PUBLISH_HEADER`

File: views/editor/publish.html [hunk 5]
```diff
@@ -41,4 +41,4 @@
     <div id="publish-button-publish">{{ gettext("publishBtn") }}</div>
   </div>
 </div>
-{% endblock %}
+{% endblock %}
```
Comment (gideonthomas) [inline views/editor/publish.html]: revert this change

**Comment by marcobeltempo** [comment]
@gideonthomas Thank you for all your feedback! I've got it working again on my localhost. I'll commit shortly with the revisions

**Comment by flukeout** [comment]
Looking forward to testing this. Please ping me when it's ready @marcobeltempo!

**Comment by marcobeltempo** [comment]
Everything's ready for testing @flukeout

**Comment by gideonthomas** [comment]
@marcobeltempo can you rebase this PR onto master? You seem to have some extra commits in here.

**Comment by marcobeltempo** [comment]
@gideonthomas I rebased the PR onto master

**Comment by gideonthomas** [comment]
This looks great! Thanks for the contribution @marcobeltempo!

**Comment by marcobeltempo** [comment]
@gideonthomas Thanks again for all your help! 😃
